docs/src/Build/README.md

```diff
@@ -11,39 +11,46 @@
 
 ## Account Service
 
-Account Service is a service enabling easy interaction with accounts on Frequency.
-Accounts are be defined as an `msaId` (a 64 bit identifier) and can contain additional information such as a handle, keys, and more.
+The Account Service enables easy interaction with accounts on Frequency.
+Accounts are defined as an `msaId` (64-bit identifier) and can contain additional information such as a handle, keys, and more.
```

Comment:
Should we say anything about MSA identifiers being represented as string values in the API, not actual numeric fields?

Reply:
in JSON, strings are the only safe transport for 64bit ints, so I think it is fine as is.